core/src/test/java/org/apache/struts2/interceptor/parameter/ParametersInterceptorTest.java

```diff
@@ -116,15 +116,17 @@ public void testInsecureParameters() throws Exception {
         pi.setParameters(action, vs, HttpParameters.create(params).build());
 
         // then
-        assertEquals(3, action.getActionMessages().size());
+        assertEquals(3, action.getActionErrors().size());
```

Comment:
Using Action errors instead of Action messages to communicate developer errors (only impacts DevMode)